gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-info.spec.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from '@fakers/users/users';
+import { ApisFaker } from '@gravitee/fixtures/management/ApisFaker';
+import { Visibility } from '@gravitee/management-webclient-sdk/src/lib/models';
+import faker from '@faker-js/faker';
+import apiDetails from '@pageobjects/Apis/ApiDetails';
+import { ApiImport } from '@model/api-imports';
+
+describe('API Info Page functionality', () => {
+  beforeEach(() => {
+    cy.loginInAPIM(ADMIN_USER.username, ADMIN_USER.password);
+    cy.visit('/#!/environments/default/apis/');
+    cy.url().should('include', '/apis/');
+  });
+
+  before(() => {
+    cy.log('Import (create) v2 API');
+    cy.request({
+      method: 'POST',
+      url: `${Cypress.env('managementApi')}${Cypress.env('defaultOrgEnv')}/apis/import`,
+      auth: { username: ADMIN_USER.username, password: ADMIN_USER.password },
+      body: ApisFaker.apiImport({ visibility: Visibility.PUBLIC }),
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      api = response.body;
+    });
+  });
+
+  const ApiDetails = new apiDetails();
+  const apiName = faker.commerce.productName();
+  const newName = apiName.replace(/ /g, '-');
+  const apiVersion = faker.datatype.number();
+  const apiDescription = faker.commerce.productDescription();
+  const path = require('path');
+  let api: ApiImport;
+  let apiId: ApiImport;
+
+  it('Verify Info Page Elements', () => {
+    cy.getByDataTestId('api_list_edit_button', { timeout: 5000 }).first().click();
+    cy.contains('Policy Studio').should('be.visible');
+    cy.contains('Info').should('be.visible');
+    cy.contains('Plans').should('be.visible');
```

Comment:
from my understanding `cy.contains()` is here to get the first element of the DOM containing `Policy Studio`.
It's only my opinion, but I think it's a bad practice to not be precise enough. 
If you name your API 'Policy Studio', then you don't know which element will be selected, and the test lose strength.
I would recommend to enrich that element selection with more precise criteria.
For example, here we know that `Policy Studio` is a `gio-submenu-item`, which can prefilter the selection. (as you did in ApiDetails.ts)

This comment applies to all the `cy.contains()` of the PR.

Reply:
I have adjusted these steps to use the page object as addressed above but I couldn't find any decent descriptor for things like the confirmation messages and I wouldn't expect things like "Configuration successfully saved!" to be visible anywhere else on the page.